ops/docker/op-stack-go/Dockerfile

```diff
@@ -4,17 +4,22 @@
 # TARGETARCH
 # TARGETPLATFORM
 # BUILDPLATFORM
-ARG TARGETOS
-ARG TARGETARCH
 
 # All target images use this as base image, and add the final build results.
 # It will default to the target platform.
 ARG TARGET_BASE_IMAGE=alpine:3.18
 
 # We may be cross-building for another platform. Specify which platform we need as builder.
-FROM --platform=$BUILDPLATFORM golang:1.21.3-alpine3.18 as builder
+FROM --platform=$BUILDPLATFORM golang:1.21.3-bullseye as builder
 
-RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
+# Install cross-compilation tools
+RUN apt-get update && apt-get install -y \
+    crossbuild-essential-arm64 \
```

Comment:
Can we add amd64, so arm users can target amd64?

Reply:
No longer relevant due to decision to use CGO_ENABLED=0